src/recom.jl

```diff
@@ -223,6 +228,10 @@ function recom_chain(graph::BaseGraph,
         if custom_acceptance && !satisfies_acceptance_fn(partition, acceptance_fn)
             # go back to the previous partition
             partition = partition.parent
+            # if user specifies this behavior, we do not increment the steps
+            # taken if the acceptance function fails. BEWARE - this can create
```

Comment:
This `BEWARE!` should probably be in the docstring for both chains.

Reply:
Done!